### PR TITLE
[wip] docs: extend Windows troubleshooting

### PR DESCRIPTION
**Pull request by DeerMichel**
#### Description of Change
We ran into a build system issue on Windows where `ninja` stuck in an endless loop of regenerating files - and found a way to fix it. May it be helpful for others...

cc @erickzhao @nornagon  

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/development/build-instructions-windows.md [hunk 1]
```diff
@@ -113,3 +113,8 @@ This can happen during build, when Debugging Tools for Windows has been installe
 ### ImportError: No module named win32file
 
 Make sure you have installed `pywin32` with `pip install pywin32`.
+
+### ninja stuck in endless loop of regenerating files
+
+From the `src/electron` folder run `gclient sync -D`
```
Comment (erickzhao) [inline docs/development/build-instructions-windows.md]: > From the `src/electron` folder, run...

File: docs/development/build-instructions-windows.md [hunk 2]
```diff
@@ -113,3 +113,8 @@ This can happen during build, when Debugging Tools for Windows has been installe
 ### ImportError: No module named win32file
 
 Make sure you have installed `pywin32` with `pip install pywin32`.
+
+### ninja stuck in endless loop of regenerating files
```
Comment (nornagon) [inline docs/development/build-instructions-windows.md]: Can you write a little more about what this looks like on the console / what causes this?

Comment (DeerMichel) [reply docs/development/build-instructions-windows.md]: We actually don't know what causes it 😄. Is it ok to just elaborate on how the issue looks like?

**Comment by DeerMichel** [comment]
Closing this as it's Heisenberg who tricks on us...

**Pull request closed without merging**